Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -592,6 +592,12 @@ theorem Equiv.prod_comp_finset {ι'} [DecidableEq ι] (e : ι ≃ ι') (f : ι'
 #align finset.equiv.prod_comp_finset Finset.Equiv.prod_comp_finset
 #align finset.equiv.sum_comp_finset Finset.Equiv.sum_comp_finset
 
+@[to_additive]
+theorem prod_comp_equiv [DecidableEq γ] {s : Finset α} (f : γ → β) (g : α ≃ γ) :
+    ∏ a in s, f (g a) = ∏ b in s.image g, f b := by
+  classical
+  simp [Equiv.prod_comp_finset g f rfl]
```

Comment:
Isn't this `Equiv.prod_comp`?

Reply:
Ah no, I see it isn't. Perhaps:
```suggestion
/-- For the special case when `s = univ`, see `Equiv.prod_comp` -/
@[to_additive]
theorem prod_comp_equiv (s : Finset α) (f : γ → β) (g : α ≃ γ) :
    ∏ a in s, f (g a) = ∏ b in s.map g.toEmbedding, f b := by
  classical
  simp [Equiv.prod_comp_finset g f rfl]
```

Reply:
I decided to remove this. This is already very similar to `prod_map` and `prod_image`.

Reply:
It doesn't look like you actually did remove it.

Reply:
Oops!